salt/_modules/metalk8s_kubernetes.py

```diff
@@ -351,3 +354,37 @@ def list_objects(kind, apiVersion, namespace='default', all_namespaces=False,
         raise CommandExecutionError('{}: {!s}'.format(base_msg, exc))
 
     return [obj.to_dict() for obj in result.items]
+
+
+def get_object_digest(object_key=None, checksum='sha256', *args, **kwargs):
```

Comment:
Maybe just `key` as an argument?

Reply:
Oh you're actually using a full colon-separated path on the object (I see `data:config.yaml`, when I was expecting `config.yaml`). Maybe `path` is better then.

Reply:
Hmm yes path make sense